doc/lua_api.md

```diff
@@ -4111,9 +4111,9 @@ On some specific cases, server translation could be useful. For example, filter
 a list on labels and send results to client. A method is supplied to achieve
 that:
 
-`minetest.get_translated_string(lang_code, string)`: Translates `string` using
-translations for `lang_code` language. It gives the same result as if the string
-was translated by the client.
+`minetest.get_translated_string(lang_code, S(string))`: resolves translations in the given string
+just like the client would, using the translation files for `lang_code`.
+For this to have any effect, the string needs to contain translation markup (as made by `minetest.translate`, `get_translator`, or `S()`).
```

Comment:
This text needs to be wrapped to account for the character limit per line (80 characters).
```suggestion
`minetest.get_translated_string(lang_code, S(string))`: resolves translations in
the given string just like the client would, using the translation files for
`lang_code`. For this to have any effect, the string needs to contain translation
markup (as made by `minetest.translate`, `get_translator`, or `S()`).
```